app.py

```diff
@@ -1,21 +1,28 @@
+from sklearn.manifold import TSNE
+from sklearn.cluster import KMeans
+import os
+import requests
+import pandas as pd
 from flask import Flask, jsonify
 from datasets import DownloadMode, load_dataset
 
-app = Flask(__name__)
+app = Flask(_name_)
```

Comment:
This should stay as `__name__`, right?

Reply:
it should, but for some reason, it wasn't running until I added the other one :)

Reply:
press |> in pycharm :)

Reply:
I'm using `flask app` from the terminal and it is working fine.